connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -272,6 +272,9 @@ message ExecutePlanResponse {
   // The metrics observed during the execution of the query plan.
   repeated ObservedMetrics observed_metrics = 6;
 
+  // The Spark schema
+  DataType schema = 7;
```

Comment:
@grundprinzip The actual Spark data type is necessary to rebuild the UDT objects.

Reply:
Is it a optional field only available in `df.collect`?

Reply:
Yes, it's only for `df.collect` for now.